.github/workflows/pr-actions.yml

```diff
@@ -60,6 +60,7 @@ jobs:
     needs: test-unit
     # stop the job if it runs over 20 minutes to prevent a hanging process from eating minutes
     timeout-minutes: 45
+ main
```

Comment:
This is a type leftover from the merge conflict
```suggestion
```

Reply:
Weird, after merging main into my branch via github gui, I don't see those changes reflected in my ide. Should I?

Nice catch, sorry about that. I'll merge again through github gui.